server/reports/item-usage/2_3_0/src/expiringInSixMonths.postgres.sql

```diff
@@ -13,5 +13,5 @@ FROM stock_line s
 INNER JOIN item_link i ON i.id = s.item_link_id
 INNER JOIN this_month ON true
 INNER JOIN six_months ON true
-WHERE s.expiry_date >= this_month AND s.expiry_date < six_months AND s.store_id = $storeId
+WHERE s.expiry_date < six_months AND s.store_id = $storeId
```

Comment:
The line changes here don't make sense with what was changed.

This change removes condition that stock isn't expired.
Now expiring in 6/12 months columns will also show stock which has already expired. 
This matches behaviour of OG reports